examples/point_robot.py

```diff
@@ -1,6 +1,9 @@
 import gym
 import planarenvs.pointRobot
 import numpy as np
+from sensors.sensors import PseudoSensor
+from sensors.sensors import PseudoDistSensor
+from sensors.sensors import GoalSensor
```

Comment:
Could you make them optional imports?
Just like with the obstacles. Then the imports are only performed if sensors-flag is set to 1.